### ISSUE TITLE
LauncherContainer.php, line 20: Platform Dependent Directory Separator Used

### ISSUE THREAD

**Issue by LukeLeber** [hechoendrupal/drupal-console-launcher#134]
The command formed by line 20 of LauncherContainer.php seems to be causing issues on Windows.

`
C:\Users\lal65\Sites\wcprospect8-dev>drupal
'C:\Users\lal65\Sites\wcprospect8-dev/vendor/drupal/console/bin/drupal' is not recognized as an internal or external command,
operable program or batch file.
`
Wouldn't it make sense to use DIRECTORY_SEPARATOR instead of '/'?

**Comment by LukeLeber** [comment]
Actually, after some debugging this may not be the issue.

Adding a 'drupal.bat' file to the drupal/console bin directory seemed to alleviate the problem.  Would this be an issue for the drupal/console project and not the launcher?

./vendor/drupal/console/bin/drupal.bat below (added)
`
@echo off
php "%~dp0/drupal.php" %*
`

**Comment by jmolivas** [comment]
@LukeLeber show we include this `drupal.bat` or a similar file on the repository for WinOS?, to avoid this issue.

**Comment by LukeLeber** [comment]
Hi @jmolivas,

I think that adding the file to the drupal/console would fix this for windows users.  Would you like me to make a pull request on that repo?

**Comment by jmolivas** [comment]
@LukeLeber Yes please, once we merge we will need to update the docs to reflect this feature.
